superset/utils/core.py

```diff
@@ -803,7 +803,7 @@ def zlib_compress(data):
     return zlib.compress(data)
 
 
-def zlib_decompress_to_string(blob):
+def zlib_decompress_to_string(blob, decode=True):
```

Comment:
NIT(optional): we started introducing type annotations, so when touching functions we can start adding those